plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchangeModule.java

```diff
@@ -49,7 +49,7 @@ protected void setup(Binder binder)
 
         List<URI> baseDirectories = buildConfigObject(FileSystemExchangeConfig.class).getBaseDirectories();
         if (baseDirectories.stream().map(URI::getScheme).distinct().count() != 1) {
-            throw new TrinoException(CONFIGURATION_INVALID, "Multiple schemes in exchange base directories");
+            binder.addError(new TrinoException(CONFIGURATION_INVALID, "Multiple schemes in exchange base directories"));
```

Comment:
If you changed this place you should return already. 
Going forward with some arbitrary schema makes little sense.